core/trino-main/src/main/java/io/trino/execution/resourcegroups/ResourceUsage.java

```diff
@@ -25,25 +25,29 @@ final class ResourceUsage
 {
     private final long cpuUsageMillis;
     private final long memoryUsageBytes;
+    private final long physicalDataScanUsageBytes;
```

Comment:
To be consistent with the source of this metric (`getPhysicalInputDataSize`), rename it please to something the has `physicalInput` in the name e.g. `physicalInputDataUsageBytes` or `physicalInputUsageBytes`. Here and everywhere else.